src/main/java/com/bfa/core/page/GooglePage.java

```diff
@@ -0,0 +1,75 @@
+package com.bfa.core.page;
+
+import com.bfa.core.BrowserType;
+import com.bfa.core.util.BrowserUtils;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+
+import java.util.List;
+
+/**
+ * Created by ninhdoan on 12/24/16.
+ */
+public class GooglePage extends GenericPage {
+
+   private static final String GOOGLE_URL = "https://google.com";
+
+   private static final String SEARCH_BOX_NAME = "q";
```

Comment:
Think of resource out locator type ```Name``` and locator value so that in case the UI changed, we can change both value without modifying source code.